### PR TITLE
Judge-challenge

### PR DESCRIPTION
**Pull request by angeliralves**
I add the steps that must be followed to judge a challenge.

### CODE REVIEW

File: libs/atqr/domain/src/lib/domain-spec/stories/judgeAChallenge.disl [hunk 1]
```diff
@@ -0,0 +1,14 @@
+read_also api.disl
+read_also enums.disl
+
+ 💬 judgeChallenge? :challengeId, accomplished: boolean
+
+        @challengeJudgement = ChallengeJudgement.Judged
```
Comment (GabrielaRsanches) [inline libs/atqr/domain/src/lib/domain-spec/stories/judgeAChallenge.disl]: I think the challengeJudgement has to be "Requested" , not "judged"

File: libs/atqr/domain/src/lib/domain-spec/stories/judgeAChallenge.disl [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+         @status = accomplished ? ChallengeStatus.Accomplished : ChallengeStatus.Failed
+
+        when accomplished
+          📣 ChallengeAccomplished! :challengeId
+            sendChallengeAccomplishedEmail? :challengeId
```
Comment (GabrielaRsanches) [inline libs/atqr/domain/src/lib/domain-spec/stories/judgeAChallenge.disl:11]: maybe necessary to point new challenge status

File: libs/atqr/domain/src/lib/domain-spec/stories/judgeAChallenge.disl [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+        else
+          📣 ChallengeFailed! :challengeId
```
Comment (GabrielaRsanches) [inline libs/atqr/domain/src/lib/domain-spec/stories/judgeAChallenge.disl:15]: same as above

**Review by GabrielaRsanches — CHANGES_REQUESTED** [review]

File: libs/atqr/domain/src/lib/domain-spec/stories/judgeAChallenge.disl (hunk 1 above)
Comment (svallory) [reply libs/atqr/domain/src/lib/domain-spec/stories/judgeAChallenge.disl]: actually, both happen. We first request, than the judge judges :)

**Comment by svallory** [comment]
Already implemented

**Pull request closed without merging**